### PR TITLE
Show theading configuration in INFO output

### PR DESCRIPTION
**Pull request by Arun4rangan**
https://github.com/redis/redis/pull/7446

### CODE REVIEW

File: commands/info.md [hunk 1]
```diff
@@ -305,6 +305,18 @@ For each command type, the following line is added:
 
 *   `cmdstat_XXX`: `calls=XXX,usec=XXX,usec_per_call=XXX`
 
+The **threads** section provides information about io threads.
```
Comment (itamarhaber) [inline commands/info.md:308]:
```suggestion
The **threads** section provides information about I/O threads.
```

File: commands/info.md [hunk 2]
```diff
@@ -305,6 +305,18 @@ For each command type, the following line is added:
...
+
+Here is the meaning of all fields in the section:
+
+*   `io_threads`: Maximum number of IO threads will be used
```
Comment (itamarhaber) [inline commands/info.md:312]:
```suggestion
*   `io_threads`: Maximum number of I/O threads that will be used
```

File: commands/info.md [hunk 3]
```diff
@@ -305,6 +305,18 @@ For each command type, the following line is added:
...
+*   `thread_reads_enabled`: Flag indicating if threads reading is enabled
```
Comment (itamarhaber) [inline commands/info.md:313]:
```suggestion
*   `thread_reads_enabled`: Flag indicating if reading clients queries with I/O threads is enabled
```

File: commands/info.md [hunk 4]
```diff
@@ -305,6 +305,18 @@ For each command type, the following line is added:
...
+*   `io_threads_active`: Flag indicating if io threads is active
```
Comment (itamarhaber) [inline commands/info.md:314]:
```suggestion
*   `io_threads_active`: Flag indicating if I/O threads are active
```

File: commands/info.md [hunk 5]
```diff
@@ -305,6 +305,18 @@ For each command type, the following line is added:
...
+*   `stat_io_reads_processed`: Number of read events processed by IO / Main threads
```
Comment (itamarhaber) [inline commands/info.md:315]:
```suggestion
*   `stat_io_reads_processed`: Number of read events processed by the main and I/O threads
```

File: commands/info.md [hunk 6]
```diff
@@ -305,6 +305,18 @@ For each command type, the following line is added:
...
+*   `stat_io_writes_processed`: Number of write events processed by IO / Main threads
```
Comment (itamarhaber) [inline commands/info.md:316]:
```suggestion
*   `stat_io_writes_processed`: Number of write events processed by the main and I/O threads
```

**Comment by Arun4rangan** [comment]
@itamarhaber Thanks for the review.

**Comment by Arun4rangan** [comment]
@itamarhaber I updated the PR according to the latest changes on upstream, would love to get another look. Thanks.

**Comment by itamarhaber** [comment]
Great - here's to seeing this in the wild starting with 6.0.7 and beyond :)

**Pull request merged**